Makefile

```diff
@@ -89,7 +89,7 @@ integrationtest:
 	set -e; for dir in $(ALL_GO_MOD_DIRS); do \
 	  echo "go test ./... + race in $${dir}"; \
 	  (cd "$${dir}" && \
-	    $(GOTEST) -tags=integrationtest -run=TestIntegration ./...); \
+	    EXPORT_TO_GCP=true $(GOTEST) -tags=integrationtest -run=TestIntegration ./...); \
```

Comment:
if you wanted to ensure `test` passes before running `integrationtest`, I think you could do (above): `integrationtest: test`.

Reply:
I'll leave like this for now since CircleCI is already covering that. It'll keep the build time down for the Cloud Build